test/test_simulation_execute.jl

```diff
@@ -203,20 +206,20 @@ end
     set_device_model!(template_ed, HydroEnergyReservoir, HydroDispatchReservoirBudget)
     c_sys5_hy_uc = PSB.build_system(PSITestSystems, "c_sys5_hy_uc")
     c_sys5_hy_ed = PSB.build_system(PSITestSystems, "c_sys5_hy_ed")
-    problems = SimulationProblems(
+    problems = SimulationModels(
         UC = DecisionModel(
             template_uc,
             c_sys5_hy_uc;
             optimizer = GLPK_optimizer,
-            constraint_duals = [:CopperPlateBalance],
+            constraint_duals = [ConstraintKey(CopperPlateBalanceConstraint, PSY.System)],
```

Comment:
This interface has changed so we probably want to remove all the calls `constraint_duals =` in `DecisionModel` constructors.

Reply:
This whole section is wrong and will fail to run. I'll leave it here so that we remember to fix the definition of the duals within the model.